bin/stringsCompletenessTest.py

```diff
@@ -64,27 +82,54 @@ def find_l10n_strings(file_path, l10n_properties, not_found_string_list):
             mozL10n\.get\(\"      # mozL10n.get is content code.
             ([a-zA-Z0-9_\.]*?)    # Match any valid string character
             \",.*                 # Only match mozL10n.get("...", ...)
```

Comment:
Not touched here, but `.get("...")` and `.get("...", ...)` could both be matched with a single regex that allows for either ")" or "," `/mozL10n\.get\("([...]*)"[\),]/` equivalent